src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.test.tsx

```diff
@@ -513,6 +515,72 @@ it("toggles between the product offerings in the premium upsell dialog", async (
   expect(productMontlyCta).toBeInTheDocument();
 });
 
+it("counts in Glean how ofen people click the upgrade CTA to purchase the yearly plan", async () => {
```

Comment:
```suggestion
it("counts in Glean how often people click the upgrade CTA to purchase the yearly plan", async () => {
```

Reply:
Is the title correct? It seems to say _"&hellip; purchase the **yearly** plan"_ but then below on L532-536 we seem to be selecting the **_monthly_** plan.

Reply:
Ugh... End-of-day committing, thanks for catching :facepalm: 237105aca